xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/directives.js

```diff
@@ -61,7 +60,8 @@ export const mousedownmove = {
       // Bind event listeners
       window.addEventListener("mousemove", mousemoveHandler);
       window.addEventListener("mouseup", removeMousemoveHandler);
-      window.addEventListener("focusout", removeMousemoveHandler);
+      // Stop propagation of the mousedown event
```

Comment:
Maybe explain why the propagation stop is required (and what are the consequences otherwise).

Reply:
I checked it again, and with the move I just made, it's not needed anymore.